openc3/lib/openc3/utilities/logger.rb

```diff
@@ -195,9 +195,7 @@ def log_message(severity_string, message, scope:, user:, type:, url:)
           if scope
             Topic.write_topic("#{scope}__openc3_log_messages", data)
           else
-            # The base openc3_log_messages doesn't have an associated logger
-            # so it must be limited to prevent unbounded stream growth
-            Topic.write_topic("openc3_log_messages", data, '*', 1000)
+            Topic.write_topic("NOSCOPE__openc3_log_messages", data)
```

Comment:
Do we have a good idea what is being logged without a scope?

Reply:
yes.  Primarily API calls to the cmd-tlm-api.   Also some top level operator stuff.